apps/console/src/features/users/components/add-user.tsx

```diff
@@ -78,7 +78,7 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
     ] = useState<string>(PRIMARY_USERSTORE_PROPERTY_VALUES.UsernameJavaScriptRegEx);
     const [ isUsernameRegExLoading, setUsernameRegExLoading ] = useState<boolean>(false);
     const [ password, setPassword ] = useState<string>("");
-    const [ passwordScore, setPasswordScore ] = useState<number>(-1);
+    const [ _passwordScore, setPasswordScore ] = useState<number>(-1);
```

Comment:
If we are not using this anywhere, we can remove this, right?

Reply:
Fixed with 0b225bb8b2f850c71ab6ffb8b5b75aed28d54650